apps/sms/js/thread_ui.js

```diff
@@ -2512,137 +2486,16 @@ var ThreadUI = {
 
     if (event.target.isPlaceholder) {
       typed = event.target.textContent.trim();
```

Comment:
Is it the same as ```this.recipients.inputValue``` that does trip as well? If so, maybe we can use it here instead (I admit it will be a little slower) and remove ``` if (!this.recipients.inputValue)``` from ```listContacts```?